traffic_portal/app/src/common/api/DeliveryServiceService.js

```diff
@@ -20,7 +20,7 @@
 var DeliveryServiceService = function($http, locationUtils, messageModel, ENV) {
 
     this.getDeliveryServices = function(queryParams) {
-        return $http.get(ENV.api['root'] + 'deliveryservices', {params: queryParams}).then(
+        return $http.get(ENV.api['deliveryServices'] + 'deliveryservices', {params: queryParams}).then(
```

Comment:
I think we would just update `ENV.api['root']` to 3.1, right @mitchell852? I think we would prefer to have TP using the same version for all requests if possible.

Reply:
That's a possibility, but it will mean needing to add routes to `routes.go` for 3.1 for all API routes.

Reply:
That's not the way routes work unless it was changed. If there is a single 3.1 route in existence, any handlers for 3.0 will implicitly handle 3.1 requests.